rero/rero-ils
# RECORDS: can_delete

NEW: Implement global can_delete

Signed-off-by: Aly Badr <aly.badr@rero.ch>
Co-Authored-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>

To Test:

get the PR branch
./run_tests.sh

- Nothing really to test because frontend implementation is planned in Taiga# 775 
- Interface, try to delete any document having items, you should get exception "IlsRecordError.NotDeleted"
